pic32/libraries/Wire/Wire.cpp

```diff
@@ -334,46 +336,67 @@ uint8_t TwoWire::read(void)
     if(di2c.read(&data, 1) == 1)
     {
         return((uint8_t) data);
-  }
+    }
 
-	return('\0');
+    return('\0');
```

Comment:
Thank you for fixing all of the gross indenting and braces around here. Looks so much better!